garrysmod/lua/includes/extensions/achievements.lua

```diff
@@ -30,7 +30,7 @@ function achievements.Register(name,func)
 	end
 
 	achievementsN = achievementsN + 1
-	bitsize = math.ceil(math.log(achievementsN) / math.log(2)) + 1
+	bitsize = math.ceil(math.log(achievementsN + 1, 2))
```

Comment:
Should be math.Round I'd imagine.

Reply:
Definitely not. It should be ``math.floor(math.log(achievementsN, 2)) + 1``, but I am working on an internal solution.

Reply:
Oh ye lol. 4am brain

Reply:
`math.ceil(math.log(achievementsN + 1, 2))` gives the same result as `math.floor(math.log(achievementsN, 2)) + 1` except when `achievementsN` is 0 because `log 0` is undefined.